### PR TITLE
F1 score and optimisers

### LINKED ISSUE #161: calculate the cumulative F1 score instead of F1 score

### PR DESCRIPTION
**Pull request by crangelsmith**
Fixes #161 #171 

For #161
- Separates function that calculates accuracy and f1
- Saves F1 scores for each evaluated epoch
- Improvement in aesthetics of the plots
- We produce % based confusion matrices

For #171

- Option for other optimisers

### CODE REVIEW

**Comment by marjanfamili** [comment]
shall we make the values of percentages just integer in favour of space ? 
confusion_eval

**Comment by crangelsmith** [comment]
> shall we make the values of percentages just integer in favour of space ?
> 
> int(All values *100)
> 
This has been implemetend in [454a68e](https://github.com/alan-turing-institute/affinity-vae/pull/162/commits/454a68ec9e43dadae216cbec567d34846bd69514)

**Comment by crangelsmith** [comment]
I think this is ready for review.

File: avae/config.py [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "vis_sim": VIS_SIM,
     "vis_all": False,
     "model": "a",
+    "opt_method": "adam",
```
Comment (jolaem) [inline avae/config.py:73]: We should add this to test_config as well as README

**Review by jolaem — COMMENTED** [review]
Just one comment about docs and test config. Otherwise all good!

**Review by jolaem — APPROVED** [review]

**Pull request merged**